python/nav/web/api/v1/urls.py

```diff
@@ -73,4 +73,5 @@
         name="prefix-usage-detail",
     ),
     re_path(r'^', include(router.urls)),
+    re_path(r'^refresh/$', views.JWTRefreshViewSet.as_view(), name='jwt-refresh'),
```

Comment:
Just to satisfy my own curiosity: I cannot recall why some API views are added to the DRF router instance, while some URLs are added manually to the URL config.

The noticeable difference (for me) is that URLs that are added manually like this do not show up in the official list of API endpoints (swagger/OpenAPI) when just visiting `/api/`. 

These URLs will also not be part of the versioned API.  We might need a discussion on why we have this setup and whether we should keep doing it (@hmpf )

Reply:
at this point I dont remember either :stuck_out_tongue:

Reply:
Come to think of it, I'm not convinced putting the `/refresh` endpoint at the root level of the API makes sense, at least not with such a generic name.  We already have `/token`.  How about `/token/refresh`? (or should `/token` be reserved for legacy tokens?) Maybe `/jwt/refresh` ?